src/Pages/Home.jsx

```diff
@@ -14,8 +14,13 @@ export default function Home() {
           variant="outlined"
           aria-label="outlined button group"
         >
-          <Button onClick="/login">Login</Button>
-          <Button onClick="/sign-up">Sign Up</Button>
+          <a href="/login">
+            <Button>Login</Button>
+          </a>
+          <a href="/sign-up">
+            {" "}
+            <Button>Sign Up</Button>
+          </a>
```

Comment:
If these links are for navigation you'll want to use "Link" from `react-router-dom`.

https://v5.reactrouter.com/web/api/Link